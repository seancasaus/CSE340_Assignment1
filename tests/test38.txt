pz2Ez (9)|(g) , lalcz ((((f).((U)|(9))).(e))|(C)).(((x)|(M)).((y)|(f))) , VOgPT ((R)|(3))|(o) , uq (i)|(((((D).(a)).(9)).(a)).(((J)|(j))|(S))) , PgHy7 f , oD3Ta (B)|(k) , nZra8 (((((1).(K)).(g))|(p)).(n))|((U).((((G)|(T))|(g)).((O).(l)))) , jBrls Y , pEuyV ((((X)|(S))|(h))|((((C).(2)).(m))|(o)))|(((S)|((Z)|((W)|(3))))|(w)) #
" ooYC2mf9exypnUgOlYYUGOlgffiYYk3UGOl1KgnRfCxfUgOlwf9exf3kDa9aS3Ywk1KgnBUgOlRfDa9aSf9gfgfgioiiiB9ifC2mf9f9f9YDa9aSCxyDa9aJgYfUTilooCxyf9eMfCMyfUexykfpng9i9f " 
