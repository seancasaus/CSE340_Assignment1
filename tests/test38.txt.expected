VOgPT , "o"
VOgPT , "o"
jBrls , "Y"
pEuyV , "C2m"
lalcz , "f9exy"
nZra8 , "pn"
nZra8 , "UgOl"
jBrls , "Y"
jBrls , "Y"
nZra8 , "UGOl"
pz2Ez , "g"
PgHy7 , "f"
PgHy7 , "f"
uq , "i"
jBrls , "Y"
jBrls , "Y"
oD3Ta , "k"
VOgPT , "3"
nZra8 , "UGOl"
nZra8 , "1Kgn"
VOgPT , "R"
PgHy7 , "f"
lalcz , "Cxf"
nZra8 , "UgOl"
pEuyV , "w"
lalcz , "f9exf"
VOgPT , "3"
oD3Ta , "k"
uq , "Da9aS"
VOgPT , "3"
jBrls , "Y"
pEuyV , "w"
oD3Ta , "k"
nZra8 , "1Kgn"
oD3Ta , "B"
nZra8 , "UgOl"
VOgPT , "R"
PgHy7 , "f"
uq , "Da9aS"
PgHy7 , "f"
pz2Ez , "9"
pz2Ez , "g"
PgHy7 , "f"
pz2Ez , "g"
PgHy7 , "f"
pz2Ez , "g"
uq , "i"
VOgPT , "o"
uq , "i"
uq , "i"
uq , "i"
oD3Ta , "B"
pz2Ez , "9"
uq , "i"
PgHy7 , "f"
pEuyV , "C2m"
PgHy7 , "f"
pz2Ez , "9"
PgHy7 , "f"
pz2Ez , "9"
PgHy7 , "f"
pz2Ez , "9"
jBrls , "Y"
uq , "Da9aS"
lalcz , "Cxy"
uq , "Da9aJ"
pz2Ez , "g"
jBrls , "Y"
PgHy7 , "f"
ERROR
